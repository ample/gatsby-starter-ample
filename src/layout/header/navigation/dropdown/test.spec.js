import React from "react"
import renderer from "react-test-renderer"

import { Dropdown } from "./"

describe("Dropdown", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Dropdown label="Trigger" items={[]}></Dropdown>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
