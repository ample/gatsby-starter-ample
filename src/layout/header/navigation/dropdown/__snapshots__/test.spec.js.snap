// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown renders correctly 1`] = `
<span
  className="dropdown"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  role="button"
  tabIndex={0}
>
  <div
    className="dropdown_trigger"
  >
    Trigger
    <svg
      height={8}
      viewBox="0 0 11 8"
      width={11}
    >
      <path
        d="M9.524.724l1.262 1.263L5.393 7.38 0 1.987 1.263.724l4.13 4.131z"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <ul
    className="dropdown_menu"
  />
</span>
`;
